apache/trafficcontrol
# Update TO for minimum TLS version

## What does this PR (Pull Request) do?

This update allows an administrator to set minimum TLS version for Traffic Ops.

- [x] This is not related to any Issue 

## Which Traffic Control components are affected by this PR?

- Documentation
- Traffic Ops

## What is the best way to verify this PR?

- Install TO
- Run `docker run -it drwetter/testssl.sh:latest -p <fqdn_of_traffic_ops>` and note the TLS versions supported
  ```
   Testing protocols via sockets except NPN+ALPN 

   SSLv2      not offered (OK)
   SSLv3      not offered (OK)
   TLS 1      offered (deprecated)
   TLS 1.1    offered (deprecated)
   TLS 1.2    offered (OK)
   TLS 1.3    not offered and downgraded to a weaker protocol
   NPN/SPDY   h2, http/1.1 (advertised)
   ALPN/HTTP2 h2, http/1.1 (offered)
  ```
- Set `"min_tls_version": 770` in the `traffic_ops_golang` section of cdn.conf
- restart TO
- Run `docker run -it drwetter/testssl.sh:latest -p <fqdn_of_traffic_ops>` and note that TLS version 1 is now off
  ```
   Testing protocols via sockets except NPN+ALPN 

   SSLv2      not offered (OK)
   SSLv3      not offered (OK)
   TLS 1      not offered
   TLS 1.1    offered (deprecated)
   TLS 1.2    offered (OK)
   TLS 1.3    not offered and downgraded to a weaker protocol
   NPN/SPDY   h2, http/1.1 (advertised)
   ALPN/HTTP2 h2, http/1.1 (offered)
  ```
## The following criteria are ALL met by this PR
